### ISSUE TITLE
Autocorrect with Spacebar Doesn't Work in Firefox or Brave Address Bars

### ISSUE THREAD

**Issue by HippieMitch** [futo-org/android-keyboard#344]
While using the latest 0.1.23.1 version (also the version prior) of the FUTO Keyboard I am unable to get autocorrect to function automatically with spacebar while typing in web browser address bars (tested with Firefox and Brave).  I have verified that the functionality works as expected outside of the Brave and Firefox web browser's address bars.  This functionality exists in GBoard and Swiftkey and appears to be the expected behavior.

**Comment by abb128** [comment]
I don't get autocorrect in address bars when testing with Gboard, are you sure? Generally URL bars are a place where you don't want autocorrect to interfere and potentially take you to the wrong website. The Chromium address bar actually explicitly requests suggestions to not even be shown

**Comment by HippieMitch** [comment]
You're right, GBoard doesn't do that either.  I'm not referring to typing in a web address so much as when typing a search query.  I think if there's a space maybe it could switch back on autocorrect as it is likely a search query at that point?  Not sure how practical that would be.

**Comment by dean-futo** [comment]
Consolidated into #666

**Comment by dean-futo** [comment]
Duplicate of #452 - see that issue for further details.
